evadb/utils/generic_utils.py

```diff
@@ -489,6 +489,16 @@ def try_to_import_qdrant_client():
         )
 
 
+def try_to_import_pinecone_client():
+    try:
+        import pinecone  # noqa: F401
+    except ImportError:
+        raise ValueError(
+            """Could not import pinecone_client python package.
+                Please install it with 'pip install pinecone_client`."""
```

Comment:
nit typo pinecone-client

Reply:
Actually pinecone_client also installs the same package